packages/components/tag/src/tag.vue

```diff
@@ -66,4 +72,9 @@ const handleClose = (event: MouseEvent) => {
 const handleClick = (event: MouseEvent) => {
   emit('click', event)
 }
+
+const handleVNodeMounted = (vnode: VNode) => {
+  // @ts-ignore
+  vnode.component.subTree.component.bum = null
```

Comment:
This operation of modifying the internal properties of a component may be adjusted in the future. Do you think we should keep this operation? @tolking

Reply:
The probability of changing `bum(onBeforeUnmount)` is very small. 

We can fix the issues through this PR before finding a more suitable solution